fdi/findings_data_import.py

```diff
@@ -199,10 +195,17 @@ def import_data(
                         finding[field_map_dict[field]] = finding[field]
                     finding.pop(field, None)
 
+            # Get RVA ID from last 4 digits
+            rvaId = re.search(r"(\d{4})(?:\.\d{1,})?$", finding["RVA ID"])
```

Comment:
_Code never lies, comments sometimes do._
–Ron Jeffries

This comment does not match the function of the regular expression that follows.
See: https://regexr.com/4p7pc

Please rewrite to include the information in the `README.md`